test/scenarios/activegate/README.md

```diff
@@ -0,0 +1,16 @@
+# Prerequisites
+k8s cluster
+
+# Setup
+CloudNative deployment with CSI driver
+
+# Goals
+Verification if ActiveGate is rolled out successfully. All AG capabilities are enabled in Dynakube.
+
+Tests:
```

Comment:
we should split this into a numbered list of steps and then a sub list with a rough outline of checks done for each

example:
1. create Activegate
  - verify pod is running
  - ...
2. ...